src/index.css

```diff
@@ -0,0 +1,31 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
+  font-family:
+    source-code-pro,
+    Menlo,
+    Monaco,
+    Consolas,
+    'Courier New',
+    monospace;
+}
```

Comment:
- [ ] As per the project requirements, you are required to delete all the boilerplate text, images, and styles that the CRA created.

![image](https://user-images.githubusercontent.com/86318284/189194828-4803a7d7-47dc-4306-

Reply:
I appreciate your effort and time.
I resolved the issue, thanks again @Nemwel-Boniface